config/config.txt

```diff
@@ -559,6 +559,24 @@ DEFAULT_VIEW_SQUARE 15x15
 ## Once a typepath causes overrun from hard deletes this many times, stop hard deleting it on garbage collection failures. (set to 0 to disable)
 #HARD_DELETES_OVERRUN_LIMIT 0
 
+## The urgent ahelp cooldown for a given player if they're alone on a server and need to send an urgent ahelp.
+URGENT_AHELP_COOLDOWN 300
+
+## The message that is sent to the discord if an urgent ahelp is sent. Useful for sending a role ping.
+#URGENT_AHELP_MESSAGE Ban ban ban ban
+
+## The message that the player receives whenever prompted whether they want to send an urgent ahelp or not.
+#URGENT_AHELP_USER_PROMPT This'll ping the admins!
+
+## The URL to the webhook for adminhelps to relay the urgent ahelp message to
```

Comment:
```suggestion
## The URL to the webhook for adminhelps to relay the urgent ahelp message to.
## If not set, will disable urgent ahelps.
```

I think you should also put this one above the rest, only because this looks like the important ones for server hosts to look at more than the others if they want to disable.

Reply:
Should also be clear that it's formatted as a Discord webhook, it's not obvious from just "webhook".

Reply:
it doesn't have to be a discord webhook, the data is pretty generic but sure